docusaurus.config.js

```diff
@@ -207,6 +219,7 @@ module.exports = {
           // Please change this to your repo.
           editUrl:
             "https://github.com/camunda/camunda-platform-docs/edit/main/",
+          beforeDefaultRemarkPlugins: [versionedLinks],
```

Comment:
🔗 This configures the _main_ docs to use the MDX plugin that interprets link target URLs based on the source file directory.